src/modules/vault/components/box/index.tsx

```diff
@@ -74,8 +76,19 @@ const VaultBox = (props: VaultBoxPropx) => {
 
   const {
     screenSizes: { isMobile },
+    authDetails: {
+      userInfos: { network },
+    },
   } = useWorkspaceContext();
 
+  const redirectToNetwork = () =>
```

Comment:
On file `src/modules/network/services/methods.ts` 
add on object of array `availableNetWorks` a new key `explorer`

and on this file get by `findBlockExplorerByNetwork(chainUrl)`, use chain by userAuth or localStorage track

Reply:
done